doc/user_guide/jupyter_chart.rst

```diff
@@ -0,0 +1,403 @@
+    should include either a ``fields`` or ``encodings`` argument, which will result in the
+    ``JupyterChart`` containing a ``PointSelection`` rather than an ``IndexSelection``.
+
+
+Interval Selections
+~~~~~~~~~~~~~~~~~~~
+The ``IntervalSelection`` class is used to store the current state of a Vega-Altair interval selection
+(as created by ``alt.selection_interval()``). In this case, the ``value`` property of the selection
+is a dictionary from column names to selection intervals
+
+.. code-block:: python
+
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.cars()
+    brush = alt.selection_interval(name="interval")
+
+    chart = alt.Chart(source).mark_point().encode(
+        x='Horsepower:Q',
+        y='Miles_per_Gallon:Q',
+        color=alt.condition(brush, 'Cylinders:O', alt.value('grey')),
+    ).add_params(brush)
+
+    jchart = alt.JupyterChart(chart)
+    jchart
+
+
+.. raw:: html
+
+    <video controls>
+      <source src="https://github.com/altair-viz/altair/assets/15064365/8b1b6ea6-6d9d-4a53-aac9-3e120609d99d">
+      Your browser does not support the video tag.
+    </video>
+
+
+Observing Selections
+~~~~~~~~~~~~~~~~~~~~
+As with variable parameters, it's possible to register a callback function to be invoked
+when a selection changes by using the ``observe`` method on the ``selections`` property.
+Here is an example that listens for changes to an interval selection, then uses the selection
+value to filter the input DataFrame and display it's HTML representation. An ipywidgets ``VBox``
+is used to combine the chart and HTML table in a column layout.
+
+.. code-block:: python
+
+    import ipywidgets
+    from IPython.display import display
+    from ipywidgets import HTML, VBox
+
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.cars()
+    brush = alt.selection_interval(name="brush")
+
+    chart_widget = alt.JupyterChart(alt.Chart(source).mark_point().encode(
+        x='Horsepower:Q',
+        y='Miles_per_Gallon:Q',
+        color=alt.condition(brush, 'Cylinders:O', alt.value('grey')),
+    ).add_params(brush))
+
+    table_widget = HTML(value=source.iloc[:0].to_html())
+
+    def on_select(change):
+        sel = change.new.value
+        if sel is None or 'Horsepower' not in sel:
+            filtered = source.iloc[:0]
+        else:
+            filtered = source.query(
+                f"{sel['Horsepower'][0]} <= `Horsepower` <= {sel['Horsepower'][1]} & "
+                f"{sel['Miles_per_Gallon'][0]} <= `Miles_per_Gallon` <= {sel['Miles_per_Gallon'][1]}"
+            )
+        table_widget.value = filtered.to_html()
+
+    chart_widget.selections.observe(on_select, ["brush"])
+
+    VBox([chart_widget, table_widget])
+
+.. raw:: html
+
+    <video controls>
+      <source src="https://github.com/altair-viz/altair/assets/15064365/1414ed59-8782-4c2e-8dd3-905d2f85fa7e">
+      Your browser does not support the video tag.
+    </video>
+
+Limitations
```

Comment:
I miss setting `setting` value of `selections` here. I often use the `init` argument to start with a certain selection.

Can we start and change the value of a bound legend selection?

As far as a I understand this is not possible (yet?)

Reply:
Hm, yes, not so straightforward.. referencing https://github.com/vega/vega-lite/issues/1830 for more info why this is still hard.